pkg/monitor/querier.go

```diff
@@ -100,6 +111,26 @@ func (monitor *agentMonitor) GetLocalPodNum() int32 {
 	return int32(monitor.interfaceStore.GetContainerInterfaceNum())
 }
 
+func (monitor *agentMonitor) GetAgentConditions() []v1beta1.AgentCondition {
+	lastHeartbeatTime := metav1.Now()
+	connectionStatus := v1.ConditionTrue
+	if !monitor.networkPolicyInfoQuerier.GetAgentControllerConnectionStatus() {
+		connectionStatus = v1.ConditionFalse
+	}
+	return []v1beta1.AgentCondition{
+		{
+			Type:              v1beta1.AgentHealthy,
+			Status:            v1.ConditionTrue,
+			LastHeartbeatTime: lastHeartbeatTime,
```

Comment:
So, must we duplicate the heartbeatTime for every condition?

Reply:
Thanks for pointing out. You are right and it would be better to only populate it once. I have updated this part and it will only be carried by type Agent/ControllerHealthy now. I also change octant-plugin a little to filter the type in order to populate LastHeartbeatTime on UI. Please help take a look.

Reply:
Let me take a further look at this part. I tested this part and found that the output format with command `kubectl describe aai` is a little bit strange with nil field for other conditions. Will update later.

Reply:
@jianjuns The output for `kubectl describe aai` will have nil LastHeartbeatTime field for other Agent/ControllerCondition types in this case. If we add more types (OVSDBConnection, OpenflowConnection) later, it may look bad on command output or UI for CRD details. 

It seems that Kubernetes can also populate duplicate timestamps for different NodeCondtitionTypes (https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L1124). So I set the field for different types at this time.

Another option is that we can move lastHeartbeatTime out of Agent/ControllerCondition and make it a separate field. What do you think?

Reply:
I think it depends on how many conditions we will have. Do you see other K8s resource conditions have the lastHeartBeatTime field?
@tnqn: what you think?

Anyway we could change in a separate PR.

Reply:
K8s node has this field and it seems that they populate it for different types. https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L1133

I will merge this PR and make another PR regarding conditions for both OVSDB connection and Openflow connection. We could change in that PR.